rdrf/rdrf/utils.py

```diff
@@ -441,3 +442,164 @@ def error(text):
 
     def __repr__(self):
         return self.text
+
+
+
+class TimeStripper(object):
+    """
+    This class exists to fix an error we introduced in the migration
+    moving from Mongo to pure Django models with JSON fields ( "Modjgo" 0objects.)
```

Comment:
Also, can you please update the comment to not say this should be used from a migration.